src/main/java/de/tum/in/www1/artemis/web/rest/FileUploadExerciseResource.java

```diff
@@ -193,20 +187,17 @@ private void validateNewOrUpdatedFileUploadExercise(FileUploadExercise fileUploa
     }
 
     /**
-     * GET /courses/:courseId/exercises : get all the exercises.
+     * GET courses/:courseId/file-upload-exercises : get all the exercises.
```

Comment:
We could adjust `all the exercises` as well, but it should be clear enough in the context.